EIPS/eip-55.md

```diff
@@ -37,6 +37,13 @@ test('0xD1220A0cf47c7B9Be7A2E6BA89F429762e7b9aDb')
 
 In English, convert the address to hex, but if the `i`th digit is a letter (ie. it's one of `abcdef`) print it in uppercase if the `4*i`th bit of the hash of the lowercase hexadecimal address is 1 otherwise print it in lowercase.
 
+# Rationale
+
+Benefits:
+- Backwards compatible with many hex parsers that accept mixed case, allowing it to be easily introduced over time
+- Keeps the length at 40 characters
+- On average there will be 15 check bits per address, and the net probability that a randomly generated address if mistyped will accidentally pass a check is 0.0247%. This is a ~50x improvement over ICAP, but not as good as a 4-byte check code.
```

Comment:
If you mention ICAP, you may as well mention that it doesn't have the limitation as ICAP, which only represent 155 bits.

Reply:
This could be done in another PR.